src/components/Plants/index.tsx

```diff
@@ -1,17 +1,55 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { ServerOrganization } from 'src/types/Organization';
 import strings from 'src/strings';
 import TfMain from 'src/components/common/TfMain';
-import Title from 'src/components/common/Title';
+import { Box, Grid, Typography, useTheme } from '@mui/material';
+import { Select } from '@terraware/web-components';
+import { listPlantingSites } from 'src/api/tracking/tracking';
+import { PlantingSite } from 'src/api/types/tracking';
+import { useDeviceInfo } from '@terraware/web-components/utils';
 
 type PlantsDashboardProps = {
   organization: ServerOrganization;
 };
 
 export default function PlantsDashboard(props: PlantsDashboardProps): JSX.Element {
+  const { organization } = props;
+  const [selectedPlantingSite, setSelectedPlantingSite] = useState<PlantingSite>();
+  const [plantingSites, setPlantingSites] = useState<PlantingSite[]>([]);
+  const theme = useTheme();
+  const { isMobile } = useDeviceInfo();
+
+  useEffect(() => {
+    const populatePlantingSites = async () => {
+      const serverResponse = await listPlantingSites(organization.id);
+      if (serverResponse.requestSucceeded) {
+        setPlantingSites(serverResponse.sites ?? []);
```

Comment:
We should default to the first one - if the url does not have a planting site id in the path.
Otherwise match it to the planting site id in the url path if one exists.
We should do the same thing we do for monitoring.